src/shogun/optimization/L1PenaltyForTG.h

```diff
@@ -123,6 +127,7 @@ class L1PenaltyForTG: public L1Penalty
 	SGVector<float64_t> m_q;
 
 private:
+	/** init */
```

Comment:
No real need of Doxygen-type documentation for private methods (since they do not belong to the interface of the class). Especially when they are dummy placeholder descriptions such as this one.